clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -552,7 +555,7 @@ fatbinary(ArrayRef<std::pair<StringRef, StringRef>> InputFiles,
     CmdArgs.push_back(
         Args.MakeArgString(Twine("-compression-level=") + Arg->getValue()));
 
-  SmallVector<StringRef> Targets = {"-targets=host-x86_64-unknown-linux-gnu"};
```

Comment:
just wondering what required the change here?

Reply:
It looks like my rebasing reverted some changes unintentionally. I believe they should all have been added back.